src/index.ts

```diff
@@ -433,11 +433,16 @@ export namespace OneSignal {
     export function addTag(key: string, value: string) {
       if (!isNativeModuleLoaded(RNOneSignal)) return;
 
-      if (!key || (!value && value !== '')) {
+      if (!key || !value) {
```

Comment:
If you want to allow people to pass in `false` as the value, it will not pass this check.

Reply:
Oops! Updated to check if null or undefined.